external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/AbstractHBaseBolt.java

```diff
@@ -37,8 +37,7 @@
 public abstract class AbstractHBaseBolt extends BaseRichBolt {
     private static final Logger LOG = LoggerFactory.getLogger(AbstractHBaseBolt.class);
 
-    protected OutputCollector collector;
-
+    protected transient OutputCollector collector;
```

Comment:
Transient is for serializability errors, I am not sure why it didn't error out in earlier when people tried to use this code. 

Since Storm serializes the Bolt code for deployment this should be marked as transient.